tests/rptest/tests/acls_test.py

```diff
@@ -280,3 +281,46 @@ def test_mtls_principal(self, rules=None, fail=False):
                 assert not fail, "list acls should have failed"
             except ClusterAuthorizationError:
                 assert fail, "list acls should have succeeded"
+
+
+class AccessControlListTestUpgrade(AccessControlListTest):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.installer = self.redpanda._installer
+
+    def setUp(self):
+        super(AccessControlListTestUpgrade, self).setUp()
+        return
+
+    def check_permissions(self):
+        # run a few times for good health
+        for _ in range(5):
+            try:
+                self.get_client("base").acl_list()
+                assert False, "list acls should have failed"
```

Comment:
Can't the fetching of ACLs fail for some other reason? I would have expected the test to check for successful SASL authN before and after the upgrade.

Reply:
This is primarily checking that AuthZ is enabled (and that "base" user does not have permission to list acls).

Line 305 is checking that AuthN is working (and user "cluster_describe" has permissions)